usage-tests/cra/src/App.tsx

```diff
@@ -0,0 +1,58 @@
+import { howMuch } from "howmuchcarbon";
+import { useMemo, useState } from "react";
+import styled from "styled-components";
+import "./App.css";
+
+const App = () => {
+  const [queryString, setQueryString] = useState("");
+  const results = useMemo(() => howMuch(queryString), [queryString]);
+
+  return (
+    <AppContainer>
+      <SearchInput onChange={(e) => setQueryString(e.target.value)} />
+      <ResultsList>
+        {results.results.map(({ name, co2Eq, source }, i) => {
+          return (
+            <ResultsListElement key={i}>
```

Comment:
This `key={i}` makes me wonder if we shouldn't expose a (maybe computed) key for the results? 🤷‍♂️

Reply:
I don't know what to think about this at this stage